org.osate.ui/src/org/osate/ui/wizards/AbstractNewFileWizard.xtend

```diff
@@ -229,7 +230,15 @@ abstract class AbstractNewFileWizard extends Wizard implements INewWizard {
 			wizardPage.pageComplete = false
 		])
 	}
-			
+	
+	def protected String validateFileName(IContainer parent, String name) {
+		if (file.exists) {
```

Comment:
Added a new method validateFileName() that is invoked by validate().  This method is meant to be overridden by subclasses.  The initial implementation mimics was in validate().